app/helpers/ExerciseConfig/Pipeline/Box/Boxes/CustomCompilationBox.php

```diff
@@ -120,12 +114,15 @@ public function compile(CompilationParams $params): array
         }
         $task->setCommandArguments($args);
 
+        if ($this->getOutputPortValue(self::$BINARY_FILE_PORT_KEY)->isEmpty()) {
```

Comment:
Does this really check that there is no variable assigned to this port?

Reply:
True, will fix.